src/modal/modal.scss

```diff
@@ -66,6 +69,11 @@
         margin-right: $iui-s;
       }
     }
+
+    .iui-modal-content {
```

Comment:
I really like this idea.  If this gets implemented you should be able to apply `max-height: 100vh;` to the base `.iui-modal-dialog` and would solve #425 .

Reply:
Done. This only solves #425 if the content is wrapped in `iui-modal-content`

Reply:
This will be fixed in CSS 1.0, when flex is added to both modals.